src/ui/ManageApi.cs

```diff
@@ -87,12 +90,22 @@ public async Task SaveEnrichmentCourse(string instCode, string ucasCode, CourseE
         {
             await _apiClient.Enrichment_SaveCourseAsync(instCode, ucasCode, course);
         }
-        
-        public async Task<bool> PublishEnrichmentCourse(string ucasCode, string courseCode)
+
+        public async Task<SearchAndCompare.Domain.Models.Course> GetSearchAndCompareCourse(string ucasCode, string courseCode)
         {
-            await _apiClient.Enrichment_PublishCourseAsync(ucasCode, courseCode);
+            var result = await _apiClient.Publish_GetSearchAndCompareCourseAsync(ucasCode, courseCode);
+            var jsonCourse = JsonConvert.SerializeObject(result, _jsonSerializerSettings);
```

Comment:
It would be good to have a comment here explaining why this is happening

Reply:
Added comments